web/src/user_card_popover.ts

```diff
@@ -676,8 +677,12 @@ function register_click_handlers(): void {
         "click",
         ".sender_name, .inline-profile-picture-wrapper",
         function (this: HTMLElement, e) {
-            const $row = $(this).closest(".message_row");
             e.stopPropagation();
+            if (mouse_drag.is_drag(e)) {
+                e.preventDefault();
```

Comment:
Is the preventDefault correct here? Wouldn't that prevent copying to the Linux clipboard from selecting text?